classes/class3/cat-app/views/layouts/remove.handlebars

```diff
@@ -0,0 +1,7 @@
+<h1>Remove Cat</h1>
+
+{{#if cat}}
```

Comment:
It'd be nice to see this wrapped in some HTML. We generally try to avoid any naked content sitting on webpages.